File: fizzbuzz/fizzbuzz.c
#include <stdio.h>
#define RANGE 100

int main(){
    int num = 1;
    while(num<=RANGE){
        if(num % 15 == 0){
            printf("FIZZBUZZ\n");
        }
        else if(num % 5 == 0){
            printf("BuZZ\n");
        }
        else if (num % 3 == 0){
            printf("FIZZ\n");
        }
        else{
            printf("%d\n", num);
        }
        num++;
    }
}

File: fizzbuzz/fizzbuzz.py
def fizzbuzz(range):
    for x in range:
        if x % 15 == 0:
            print("FIZZBUZZ")
        elif x % 3 == 0:
            print ("FIZZ")
        elif x % 5 == 0:
            print("BUZZ")
        else:
            print x

fizzbuzz(range(100))
